js/modules/k6/experimental/streams/tests/testharness.js.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/resources/testharness.js b/resources/testharness.js
+index c5c375e17..57d201b57 100644
+--- a/resources/testharness.js
++++ b/resources/testharness.js
+@@ -487,7 +487,11 @@
+         this.all_loaded = false;
+         this.on_loaded_callback = null;
+         Promise.resolve().then(function() {
+-            this.all_loaded = true
++            // This is the test environment used when we run tests from Go
++            // code and with the Goja/k6 runtime. However, we don't want
++            // all tests to be marked as loaded "by default", as it would
++            // make some of the tests to not be executed at all.
++            // this.all_loaded = true
+             if (this.on_loaded_callback) {
+                 this.on_loaded_callback();
+             }
+@@ -2631,6 +2635,16 @@
+             this.set_status(status, message, stack);
+             this.phase = this.phases.HAS_RESULT;
+             this.done();
++
++						// We don't want to rely on the DOM and other browser-based
```

Comment:
nit; this are tabs, the rest of the file is with spaces.

Same for hte other part of the patch.